source/extensions/filters/http/grpc_http1_bridge/http1_bridge_filter.cc

```diff
@@ -72,6 +72,7 @@ Http::FilterDataStatus Http1BridgeFilter::encodeData(Buffer::Instance& data, boo
   } else {
     // Buffer until the complete request has been processed.
     if (do_framing_) {
+      do_framing_ = false;
```

Comment:
Could you add a comment explaining why we set this to false here? The grpc reverse filter has its own variable to guard this, but changing `do_framing_` seems fine as well